metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
npm dependency trees tend to be very deep (compared even to maven ones). Should we maybe look at incorporating some form of npm license check tooling?